[Logging Service: 67]
[2024-11-07 17:32:46.369 +0100] [INFO] [cadence.virtuoso.ade.loggingservice] - HistoryRecorderManager init {"dbBasePath": "/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky/ADC/TB_single/maestro/results/maestro"}
[2024-11-07 17:32:46.369 +0100] [INFO] [cadence.virtuoso.ade.loggingservice] - useGRPC grpc service... {"brokerAddress": "cae-europractice1.othr.de:36149", "serviceName": "DaVinciService_2817718_1730254398", "axlSession": "fnxSession3"}
[2024-11-07 17:32:46.369 +0100] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service has successfully started. Waiting for incoming messages ... {"hostname": "cae-europractice1.othr.de", "processid": 3399218, "logfile": "/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky/logs_cae3/logs0/LoggingService67.log"}
[2024-11-07 17:32:46.382 +0100] [INFO] [cadence.virtuoso.ade.loggingservice] - logging service gRPC started {"broker address": "cae-europractice1.othr.de:42775"}
[2024-11-07 22:52:19.972 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:19.972 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:20.498 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "UNIQUE constraint failed: location.point, location.test, location.tool", "sql": "INSERT INTO location (tool, file, point, test, line) VALUES ('unknown', '', '', '', '0');"}
[2024-11-07 22:52:20.498 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:20.500 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "delete from meta;"}
[2024-11-07 22:52:20.500 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('HistorySimDir','/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0');"}
[2024-11-07 22:52:20.500 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('WorkDir','/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky');"}
[2024-11-07 22:52:20.500 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "insert into meta (key, value) values('version','1');"}
[2024-11-07 22:52:20.530 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='exprOutputs.log.76.9.1' WHERE tool='Expression Evaluator' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/exprOutputs.log.76.9.1';"}
[2024-11-07 22:52:20.530 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='logs_cae3/logs0/Service75.log' WHERE tool='Netlist Service' AND file='/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky/logs_cae3/logs0/Service75.log';"}
[2024-11-07 22:52:20.530 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='logs_cae3/logs0/Job76.log' WHERE tool='Simulation Monitor' AND file='/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky/logs_cae3/logs0/Job76.log';"}
[2024-11-07 22:52:20.530 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='10/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/10/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.531 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='100/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/100/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.531 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='101/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/101/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.531 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='102/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/102/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.531 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='103/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/103/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.531 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='104/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/104/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='105/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/105/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='106/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/106/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='107/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/107/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='108/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/108/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='109/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/109/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.532 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='11/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/11/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.533 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='110/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/110/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.533 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='111/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/111/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.533 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='112/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/112/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.533 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='113/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/113/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.533 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='114/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/114/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='115/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/115/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='116/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/116/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='117/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/117/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='118/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/118/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='119/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/119/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.534 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='12/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/12/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.535 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='120/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/120/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.535 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='121/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/121/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.535 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='122/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/122/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.535 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='123/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/123/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.535 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='124/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/124/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.536 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='125/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/125/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.536 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='126/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/126/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.536 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='127/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/127/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.536 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='128/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/128/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.537 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='129/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/129/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.537 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='13/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/13/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.537 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='130/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/130/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.537 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='131/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/131/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.537 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='132/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/132/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.538 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='133/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/133/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.538 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='134/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/134/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.538 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='135/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/135/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.538 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='136/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/136/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.538 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='137/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/137/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.539 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='138/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/138/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.539 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='139/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/139/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.539 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='14/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/14/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.539 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='140/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/140/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.539 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='141/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/141/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.540 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='142/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/142/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.540 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='143/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/143/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.540 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='144/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/144/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.540 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='145/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/145/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.541 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='146/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/146/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.541 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='147/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/147/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.541 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='148/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/148/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.541 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='149/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/149/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.541 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='15/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/15/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.542 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='150/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/150/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.542 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='151/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/151/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.542 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='152/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/152/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.543 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='153/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/153/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.543 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='154/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/154/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.543 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='155/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/155/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.543 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='156/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/156/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.544 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='157/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/157/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.544 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='158/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/158/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.544 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='159/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/159/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.544 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='16/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/16/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.544 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='160/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/160/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.545 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='161/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/161/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.545 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='162/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/162/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.545 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='163/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/163/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.545 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='164/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/164/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.546 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='165/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/165/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.546 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='166/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/166/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.546 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='167/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/167/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.547 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='168/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/168/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.547 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='169/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/169/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.547 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='17/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/17/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.547 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='170/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/170/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.547 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='171/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/171/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.548 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='172/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/172/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.548 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='173/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/173/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.548 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='174/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/174/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.549 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='175/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/175/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.549 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='176/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/176/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.549 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='177/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/177/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.549 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='178/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/178/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.550 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='179/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/179/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.550 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='18/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/18/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.550 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='180/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/180/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.550 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='181/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/181/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.551 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='182/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/182/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.551 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='183/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/183/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.551 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='184/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/184/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.551 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='185/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/185/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.552 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='186/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/186/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.552 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='187/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/187/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.552 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='188/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/188/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.553 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='189/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/189/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.553 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='19/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/19/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.553 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='190/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/190/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.553 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='191/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/191/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.554 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='192/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/192/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.554 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='193/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/193/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.554 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='194/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/194/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.555 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='195/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/195/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.555 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='196/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/196/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.555 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='197/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/197/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.555 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='198/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/198/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.556 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='199/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/199/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.556 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='20/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/20/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.556 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='200/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/200/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.556 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='201/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/201/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.557 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='202/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/202/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.557 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='203/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/203/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.557 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='204/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/204/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.557 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='205/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/205/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.558 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='206/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/206/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.558 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='207/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/207/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.558 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='208/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/208/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.559 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='209/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/209/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.559 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='21/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/21/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.559 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='210/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/210/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.559 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='211/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/211/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.560 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='212/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/212/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.560 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='213/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/213/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.560 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='214/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/214/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.560 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='215/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/215/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.561 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='216/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/216/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.561 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='217/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/217/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.561 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='218/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/218/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.561 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='219/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/219/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.561 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='22/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/22/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.562 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='220/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/220/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.562 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='221/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/221/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.562 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='222/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/222/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.563 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='223/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/223/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.563 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='224/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/224/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.563 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='225/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/225/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.563 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='226/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/226/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.564 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='227/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/227/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.564 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='228/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/228/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.564 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='229/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/229/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.564 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='23/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/23/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.565 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='230/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/230/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.565 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='231/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/231/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.565 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='232/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/232/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.565 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='233/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/233/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.566 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='234/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/234/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.566 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='235/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/235/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.566 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='236/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/236/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.567 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='237/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/237/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.567 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='238/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/238/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.567 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='239/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/239/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.567 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='24/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/24/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.568 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='240/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/240/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.568 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='241/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/241/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.568 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='242/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/242/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.568 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='243/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/243/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.569 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='244/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/244/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.569 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='245/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/245/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.569 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='246/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/246/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.570 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='247/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/247/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.570 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='248/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/248/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.570 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='249/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/249/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.570 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='25/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/25/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.571 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='250/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/250/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.571 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='251/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/251/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.571 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='252/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/252/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.572 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='253/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/253/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.572 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='254/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/254/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.572 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='255/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/255/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.572 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='256/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/256/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.573 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='257/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/257/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.573 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='258/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/258/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.573 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='259/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/259/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.573 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='26/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/26/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.574 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='260/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/260/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.574 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='261/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/261/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.574 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='262/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/262/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.575 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='263/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/263/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.575 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='264/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/264/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.575 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='265/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/265/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.576 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='266/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/266/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.576 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='267/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/267/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.576 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='268/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/268/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.577 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='269/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/269/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.577 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='27/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/27/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.577 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='270/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/270/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.577 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='271/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/271/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.578 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='272/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/272/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.578 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='273/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/273/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.578 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='274/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/274/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.579 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='275/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/275/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.579 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='276/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/276/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.579 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='277/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/277/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.580 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='278/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/278/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.580 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='279/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/279/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.580 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='28/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/28/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.580 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='280/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/280/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.581 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='281/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/281/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.581 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='282/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/282/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.581 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='283/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/283/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.582 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='284/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/284/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.582 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='285/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/285/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.582 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='286/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/286/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.583 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='287/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/287/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.583 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='288/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/288/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.583 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='289/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/289/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.584 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='29/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/29/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.584 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='290/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/290/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.584 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='291/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/291/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.585 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='292/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/292/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.585 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='293/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/293/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.585 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='294/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/294/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='295/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/295/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='30/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/30/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='31/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/31/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='32/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/32/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='33/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/33/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='34/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/34/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='35/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/35/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='36/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/36/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='37/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/37/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='38/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/38/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='39/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/39/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.586 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='40/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/40/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='41/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/41/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='42/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/42/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='43/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/43/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='44/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/44/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='45/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/45/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='46/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/46/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='47/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/47/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='48/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/48/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='49/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/49/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='50/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/50/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='51/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/51/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.587 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='52/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/52/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='53/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/53/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='54/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/54/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='55/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/55/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='56/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/56/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='57/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/57/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='58/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/58/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='59/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/59/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='60/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/60/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='61/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/61/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.588 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='62/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/62/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='63/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/63/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='64/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/64/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='65/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/65/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='66/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/66/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='67/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/67/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='68/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/68/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='69/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/69/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='70/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/70/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.589 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='71/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/71/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='72/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/72/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='73/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/73/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='74/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/74/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='75/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/75/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='76/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/76/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='77/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/77/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='78/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/78/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='79/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/79/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.590 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='80/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/80/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='81/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/81/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='82/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/82/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='83/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/83/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='84/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/84/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='85/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/85/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='86/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/86/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='87/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/87/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.591 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='88/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/88/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='89/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/89/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='90/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/90/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='91/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/91/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='92/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/92/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='93/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/93/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='94/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/94/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.592 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='95/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/95/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='96/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/96/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='97/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/97/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='98/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/98/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - ServiceDBAccessSqliteOp::evalSql failed {"reason": "attempt to write a readonly database", "sql": "UPDATE location SET file='99/ADC_TB_single_1/psf/spectre.out' WHERE tool='spectre' AND file='/home/cae3/simulation/ADC/TB_single/maestro/results/maestro/ExplorerRun.0/99/ADC_TB_single_1/psf/spectre.out';"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:20.593 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:21.732 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert location failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:21.732 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-07 22:52:22.049 +0100] [ERROR] [cadence.virtuoso.ade.loggingservice] - historySqliteRecorder::insert failed {"reason": "attempt to write a readonly database"}
[2024-11-08 01:45:28.545 +0100] [PROCESS] [cadence.virtuoso.ade.loggingservice] - Logging Service stop!!!! {"hostname": "cae-europractice1.othr.de", "processid": 3399218, "logfile": "/home/cae3/Desktop/ADC/ASIC_ADC/Cadence_sky/logs_cae3/logs0/LoggingService67.log"}
